### ISSUE TITLE
#allowBreak shouldn't affect #mine

### ISSUE THREAD

**Issue by SuperOP535** [cabaletta/baritone#377]
Yeah

**Comment by leijurv** [comment]
this is theoretically possible with a custom CalculationContext passed from MineProcess I guess...

**Comment by LoganDark** [comment]
Yeah, I should be able to mine exposed ores without Baritone tunneling through stone, right?
